racingcar-java/src/main/java/step3/domain/Cars.java

```diff
@@ -27,4 +26,22 @@ public void move() {
                 .peek(Car::move)
                 .collect(Collectors.toList());
     }
+
+    public List<Car> winner(){
+        int max = maxCarPoint();
+
+        return cars.stream()
+                .filter(car -> max == car.getDistance())
+                .collect(Collectors.toList());
+    }
+
+    private int maxCarPoint(){
+        Car car = cars
+                .stream()
+                .max(Comparator.comparing(Car::getCarName))
+                .orElse(new Car("test",0));
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalStateException("error"));
```
찾을 수 없다면 특정 잘못된 객체를 반환하기보다는 예외로 처리해주세요

Reply:
아 그러는편이 좋겠네요 감사합니다 :)